release/release_workflow.wdl

```diff
@@ -42,137 +33,324 @@ task do_release {
 
-        CROMWELL_UPLOAD_URL=$(sed 's/{.*}/?name=cromwell-${newVersion}.jar/' <<< "$UPLOAD_URL")
-        WOMTOOL_UPLOAD_URL=$(sed 's/{.*}/?name=womtool-${newVersion}.jar/' <<< "$UPLOAD_URL")
+        CROMWELL_UPLOAD_URL=$(sed 's/{.*}/?name=~{cromwellJarName}/' <<< "$UPLOAD_URL")
+        WOMTOOL_UPLOAD_URL=$(sed 's/{.*}/?name=~{womtoolJarName}/' <<< "$UPLOAD_URL")
 
         # Upload the cromwell jar as an asset
-        curl -X POST --data-binary @${cromwellJar} -H "Authorization: token ${githubToken}" -H "Content-Type: application/octet-stream" "$CROMWELL_UPLOAD_URL"
+        curl -X POST --data-binary @~{cromwellJar} -H "Authorization: token ~{githubToken}" -H "Content-Type: application/octet-stream" "$CROMWELL_UPLOAD_URL"
 
         # Upload the womtool jar as an asset
-        curl -X POST --data-binary @${womtoolJar} -H "Authorization: token ${githubToken}" -H "Content-Type: application/octet-stream" "$WOMTOOL_UPLOAD_URL"
+        curl -X POST --data-binary @~{womtoolJar} -H "Authorization: token ~{githubToken}" -H "Content-Type: application/octet-stream" "$WOMTOOL_UPLOAD_URL"
 
         # Publish the draft
-        curl -X PATCH -d '{"draft": false}' https://api.github.com/repos/${organization}/cromwell/releases/"$RELEASE_ID"?access_token=${githubToken}
+        curl -X PATCH -d '{"draft": false}' https://api.github.com/repos/~{organization}/cromwell/releases/"$RELEASE_ID"?access_token=~{githubToken}
     >>>
     runtime {
         docker: "python:2.7"
     }
+    output {
+        String cromwellReleaseUrl = "https://github.com/~{organization}/cromwell/releases/download/~{newVersion}/~{cromwellJarName}"
+        String womtoolReleaseUrl = "https://github.com/~{organization}/cromwell/releases/download/~{newVersion}/~{womtoolJarName}"
+    }
 }
 
-workflow release_cromwell {
-  String githubToken
-  String organization
+task releaseHomebrew {
+    input {
+        String organization
+        String githubToken
+        
+        String releaseVersion
+        
+        String cromwellReleaseUrl
+        String womtoolReleaseUrl
+        
+        File cromwellJar
+        File womtoolJar
+    }
+    
+    String branchName = "cromwell-~{releaseVersion}"
+    
+    # This is to allow for testing. If the organization is not broadinstitute, meaning we're doing a test,
+    # instead of creating the PR in the official homebrew repo make a PR in the test repo.
+    String headBranch = if (organization == "broadinstitute") then "broadinstitute:~{branchName}" else branchName
+    String baseBranch = "master"
+    String pullRepo = if (organization == "broadinstitute") then "Homebrew" else organization
 
-  String cromwellTemplate = "sed -i '' \"s/cromwellVersion[[:space:]]=.*/cromwellVersion = \\\"<<VERSION>>\\\"/g\" project/Version.scala"
+    command <<<
+        set -e
+        set -x 
+
+        # Clone the homebrew fork
+        git clone git@github.com:~{organization}/homebrew-core.git --depth=100
```

Comment:
can there be some docs as to what's going on here with cloning a fork of homebrew?

Reply:
Now that I'm looking at the homebrew docs, there is a brew command that does all this automatically -_-
https://docs.brew.sh/How-To-Open-a-Homebrew-Pull-Request

Reply:
![download](https://user-images.githubusercontent.com/961771/41865936-8b3c0086-7863-11e8-983f-ebe7857eaf37.png)

Reply:
It doesn't seem like it could handle the update of womtool in the same formula, so I'm going to leave like this for this release and added a comment in the WDL to look into it.